core/trino-parser/src/main/java/io/trino/sql/tree/Property.java

```diff
@@ -20,25 +20,46 @@
 import java.util.Optional;
 
 import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkState;
 import static java.util.Objects.requireNonNull;
 
+// TODO: Model DEFAULT as a type of expression to make it easier to do analysis, planning and error reporting in a uniform way
```

Comment:
Please add a Trino issue for that and reference it here

Reply:
Created issue https://github.com/trinodb/trino/issues/10687.